### ISSUE TITLE
Why do I need to right-click a box to select it?

### ISSUE THREAD

**Issue by xvolks** [ryouchinsa/Rectlabel-support#12]
I often have to modify boxes copied with command + i from the previous image.
Usually I just have to move each boxes, add/delete one or two boxes and few resizes.
To select a box I have to right click it, the contextual menu pops in, left click outside the menu to discard it, click the center point to move the box, done.

Why can't I just left click the box to select it and move it? 
Is there a better solution than right click? 
Using the list of boxes on the right is not an option for me since I have many boxes with the same class name. I use this feature only when a bigger box do not let me select a smaller box that is inside the big one.

**Comment by ryouchinsa** [comment]
There are 2 modes of "Create box" and "Move" to select and move the box.
As default, you are in the "Create box" mode.

In the "Create box" mode, you can draw a box in the another box so that you cannot select the box by left click except the center point.
In the "Move" mode, you can select and move the box by left click.
To change the mode, click "Move" button or use the "command + m" shortcut.
https://rectlabel.com/help

It might be helpful if you could see which mode you are in.

To delete the box, currently you have to use right click.
We might need a shortcut key to delete a box.

**Comment by ryouchinsa** [comment]
Added short cut key "delete" to delete the box
Added short cut key "command + D" to duplicate the box
Show which mode you are in by the button color

![3](https://user-images.githubusercontent.com/1954306/29012225-d7794ac2-7b74-11e7-98e4-0e9074251aa2.png)

![2](https://user-images.githubusercontent.com/1954306/29012220-cd61e5f8-7b74-11e7-802e-0456d1f20fc2.png)

We submitted the new update to Apple.
When the new update is released, we will let you know.
Thank you.
